cli/main.py

```diff
@@ -190,16 +199,16 @@ def lookup(self, settings_files, settings_dict):
         other settings files.
         """
 
-        cli.yamls.Lookup.settings = utils.generate_settings(settings_files)
+        yamls.Lookup.settings = utils.generate_settings(settings_files)
         # todo(yfried) remove this line after refactor
-        cli.yamls.Lookup.settings.merge(settings_dict)
-        cli.yamls.Lookup.settings = utils.merge_extra_vars(
-            cli.yamls.Lookup.settings,
+        yamls.Lookup.settings.merge(settings_dict)
```

Comment:
Oleksii, the 'merge' method that being used here is the from the Configuration class. It returns the merging result...
Please assign the return value to the same calling object:
yamls.Lookup.settings = yamls.Lookup.settings.merge(settings_dict)